### DISCUSSION TITLE
Mercado Sincero

### DISCUSSION THREAD

**Discussion by vish-blog** [vish-blog/comments#434] [Imagens]
Post de referência; https://www.vish.blog.br/imagens/tmercado-sincero/

**Comment by brunoaugusto** [comment]
Eu já comi algumas "veganisses" e, realmente, o sabor era bem próximo do original, mas, pra mim, falar que tá cozinhando um bife de jaca já é absurdo

**Comment by one-eyed-willy** [comment]
Se tiver gosto de carne eu como até sacolinha de mercado :rofl:

**Comment by angellisy** [comment]
> [Willy Caolho](</perfil/?user=one-eyed-willy>) disse [segunda-feira, 20 de junho de 2022](<https://www.vish.blog.br/imagens/mercado-sincero/#comment_1160646649>):

> Se tiver gosto de carne eu como até sacolinha de mercado 🤣

[![null](<https://y.yarn.co/5fff06d2-b7c4-4ccc-9179-4196951b53ff_screenshot.jpg>)](<https://y.yarn.co/5fff06d2-b7c4-4ccc-9179-4196951b53ff.mp4>)
